### PR TITLE
Added code comments.

### PR DESCRIPTION
**Pull request by HipsterDave**

### CODE REVIEW

File: run_cube.py [hunk 1]
```diff
@@ -10,13 +10,16 @@
 import CrabRobotLibrary
 
 def run_table_3(crabot):
+    #drives to the big boccia ball
     crabot.gyro_reset()
     crabot.gyro_drive(500, 0, 130)
     crabot.gyro_turn(50, Direction.CLOCKWISE)
     crabot.gyro_drive(10000, 50, 1650)
     crabot.brake()
+    #pushes down cubes
```
Comment (ZacKeepers) [inline run_cube.py]: I think this line should be changed to, dumps out the cubes.

File: run_cube.py [hunk 2]
```diff
@@ -10,13 +10,16 @@
 import CrabRobotLibrary
 
 def run_table_3(crabot):
+    #drives to the big boccia ball
```
Comment (ZacKeepers) [inline run_cube.py]: I think we should add the word area to the end of the comment.

File: run_cube.py (hunk 1 above)
Comment (ZacKeepers) [reply run_cube.py]:
```suggestion
    #Dumps out the cubes
```

File: run_cube.py (hunk 2 above)
Comment (ZacKeepers) [reply run_cube.py]:
```suggestion
    #drives to the big boccia ball area
```

**Review by ZacKeepers — APPROVED** [review]

**Review by jkutner — APPROVED** [review]

**Pull request merged**